server-utils/Cargo.toml

```diff
@@ -16,7 +16,7 @@ globset = "0.4"
 jsonrpc-core = { version = "14.0", path = "../core" }
 lazy_static = "1.1.0"
 log = "0.4"
-tokio = { version = "0.1" }
+tokio = { version = "0.1.15" }
```

Comment:
Yes, but we don't really require the latest. I just tested that `0.1.15` is enough to fix the `Send + Sync` issue, so that's the minimal version we support.

Reply:
All right, cool I was confused by the `commit message` then

Reply:
Ah, yeah. Good point :) I wasn't paying that much attention when writing it.